aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/Translator.java

```diff
@@ -110,13 +114,17 @@ static ResourceModel translateFromSdk(
                 .build();
     }
 
-    public static DescribeDbEngineVersionsRequest describeDbEngineVersionsRequest(final ResourceModel model,
+    public static DescribeDbEngineVersionsRequest describeDbEngineVersionsRequest(final ResourceModel model, final Collection<String> engines,
                                                                                   final String nextToken) {
         return DescribeDbEngineVersionsRequest.builder()
                 .engine(model.getEngine())
                 .engineVersion(model.getEngineVersion())
-                .marker(nextToken)
+                .filters(Filter.builder()
+                        .name(ENGINE_FILTER_NAME)
+                        .values(engines)
+                        .build())
                 .includeAll(true)
```

Comment:
`Specifies whether to also list the engine versions that aren't available. The default is to list only available engine versions.`